addons/l10n_latam_check/views/account_payment_view.xml

```diff
@@ -0,0 +1,216 @@
+                        attrs="{'invisible': ['|', ('state', '!=', 'posted'), ('payment_method_code', '!=', 'new_third_party_checks')]}">
+                    <field name="l10n_latam_check_current_journal_id"/>
+                    <span attrs="{'invisible': [('l10n_latam_check_current_journal_id', '!=', False)]}">Not in Wallet</span>
+                    <button name="button_open_check_operations" type="object" string="⇒ Check Operations" class="oe_link"/>
+                </div>
+            </field>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_account_third_party_check_operations_tree">
+        <field name="name">account.check.operations.tree</field>
+        <field name="model">account.payment</field>
+        <field name="priority" eval="99"/>
+        <field name="arch" type="xml">
+            <tree default_order="date desc, id desc, name desc">
+                <field name="date"/>
+                <field name="name"/>
+                <field name="payment_type"/>
+                <field name="journal_id"/>
+                <field name="partner_id" string="Customer"/>
+                <field name="state" invisible="1"/>
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_account_check_calendar">
+        <field name="name">account.check.calendar</field>
+        <field name="model">account.payment</field>
+        <field name="arch" type="xml">
+            <calendar
+                    mode="month"
+                    date_start="l10n_latam_check_payment_date"
+                    color="journal_id">
+                <field name="amount"/>
+            </calendar>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_account_check_pivot">
+        <field name="name">account.check.calendar</field>
+        <field name="model">account.payment</field>
+        <field name="arch" type="xml">
+            <pivot>
+                <field name="l10n_latam_check_payment_date" type="row" interval="month"/>
+                <field name="l10n_latam_check_payment_date" type="row" interval="week"/>
+            <field name="amount" type="measure"/>
+            </pivot>
+        </field>
+    </record>
+
+ <!-- Own Check Views and menus -->
+
+    <record model="ir.ui.view" id="view_account_own_check_tree">
+        <field name="name">account.check.tree</field>
+        <field name="model">account.payment</field>
+        <field name="priority">100</field>
+        <field name="mode">primary</field>
+        <field name="inherit_id" ref="account.view_account_payment_tree"/>
+        <field name="arch" type="xml">
+            <field name="name" position="attributes">
+                <attribute name="optional">hide</attribute>
+            </field>
+            <field name="name" position="after">
+                <field name="check_number"/>
+            </field>
+            <field name="payment_method_line_id" position="replace"/>
```

Comment:
Removing it might break inheritance of the main view that uses this field.
Maybe put it as invisible instead?